### DISCUSSION TITLE
My 7 submissions for Sorobanathon NFTs

### DISCUSSION THREAD

**Discussion by GeekMom51** [stellar/sorobanathon#127] [General]
Here are my 7 submissions for the Sorobanathan for artists NFTs.

The Journey Begins
https://litemint.com/items/GCTXZRXKWS2AHFPDSGBPNX53J5WH7A2GLR7DEFDP3UTDURTN7JOD66QV/SOROBAN00

The Temple
https://litemint.com/items/GAW4RKFSKDQQRTZI2AT2D4S42VH62QB5D4JCJMCDLKZLY2FWG5AQGJYW/SOROBAN01

Entrance Hall
https://litemint.com/items/GCGOGD6YXZIE4IEASJ3W27TOWH65CXLKEFVL3HKV4EAW6PCE2IY244GY/SOROBAN02

Lava Snakes
https://litemint.com/items/GC2AF7HA2ZPKNAMBTRP6DK4QUL2JMO2RXFQOU5DFA2NR7YBIPEC5KI4D/SOROBAN03

The Circular Room
https://litemint.com/items/GD2VCT6FPH2NPTKB7OIDRRIDF6UNMU475S7HBMCOVUWK6ZGWJJLFIF5L/SOROBAN04

The Dark Cavern
https://litemint.com/items/GAN6ODNMDTVRNJCT6GN6E7ZLTZDXTL65X3UO6TWFFY5EXZNV4Z5WHBIS/SOROBAN05

The Golden Soroban
https://litemint.com/items/GDZMO2MV6UFJDUXUZV72ILKCRHDALSD2YVQQZKEX7ALGATAPRA4YEMUM/SOROBAN06

Thanks,
JJ

**Comment by stellarjohn** [comment]
Hi [GeekMom51](https://github.com/GeekMom51)!

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John

**Reply by GeekMom51** [reply]
Thank you! I have completed the form & kyc
